projects/debouncer/inc/debouncer.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
```

Comment:
Why is this a project rather than part of either Driver Controls or ms-helper?

Reply:
Ah, yeah it could be moved to driver controls.

Reply:
Titus told me to wait on that.

Reply:
We'll probably move it to ms-helper in the off chance that any other boards have inputs. I suppose you could make the library yourself, although one of my PRs already has it.

`make new LIBRARY=ms-helper` will create a new library. You then need to modify `rules.mk` to include `ms-helper` as a library.